hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/TestHelpers.java

```diff
@@ -828,6 +829,19 @@ public static org.apache.tuweni.bytes.Bytes bytesForRedirect(final byte[] subSel
                 org.apache.tuweni.bytes.Bytes.of(subSelector));
     }
 
+    public static org.apache.tuweni.bytes.Bytes bytesForRedirectAccount(
```

Comment:
Why not break with convention and add a one-line comment here and at L837 saying what these methods do? (No need IMO for full javadoc.)